### PR TITLE
Introduction changes

### PR DESCRIPTION
**Pull request by AnthonyLim23**
I have moved the paragraph about FAIR data and university turnover into the paragraph on community developers. This should improve the flow of the paper.

### CODE REVIEW

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 1]
```diff
@@ -109,16 +109,22 @@ \section{Introduction}
...
+With the move towards FAIR (Findable, Accessible, Inter-operable and Reusable) \cite{panosc} data, concerns about the reproducibility and reliability of data has become even more important to scientists.
+These concerns are in part rooted in the rapid replacement of developers, in the form of PhD students and post-docs.
+These developers may work in isolated conditions with limited knowledge of previous changes and the reasoning for them.
+A potential pitfall with this isolated approach of software development is that bugs are less likely to be found, due to fewer users.
+Also the software may become too cumbersome to develop leading to the project having to start from scratch.
+These factors make it more difficult to share software and for other research groups to adapt the software for their own purpose.
+However, some academic groups and research infrastructures are now starting to follow guidelines when writing software.
+Alternatively, users will depend on software created by the user community or the hosting facilities.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:121]: This sentence feels a bit random in the rest of the paragraph.

Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:121]: shall we just delete it?

Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:121]:
```suggestion
```

**Comment by AnthonyLim23** [comment]
is this good to go?

**Pull request merged**